driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java

```diff
@@ -120,31 +124,33 @@ public void run() {
 
         Executor initExecutor = manager.cluster.manager.configuration.getPoolingOptions().getInitializationExecutor();
 
-        ListenableFuture<List<Void>> allConnectionsFuture = Futures.allAsList(connectionFutures);
+        ListenableFuture<List<Void>> allConnectionsFuture = Futures.successfulAsList(connectionFutures);
```

Comment:
The goal of `wasConnectionInitialized` in the conviction policy was really to avoid decrementing the count.
We could still mark the node down if the count is 0. The only problem is that we would signal multiple times if all pooled connections fail to initialize, but the code already handles that (in fact, that's what happens with the current version).